README.md

```diff
@@ -508,22 +508,25 @@
     [abstraction](https://en.wikipedia.org/wiki/Abstraction_(software_engineering)), and
     [encapsulation](https://en.wikipedia.org/wiki/Encapsulation_(computer_programming)).
 
-* **What is the difference between a constructor and a method?** 
+* **What is the difference between a constructor and a method?**
+    - The name of the constructor is same as that of the class name, whereas the name of the method can be anything.
+    - There is no return type of a constructor.
+    - When you make an object of a class, then the constructor of that class will be called automatically. 
+      But for methods, we need to call it explicitely.
+    - Constructors can't be inherited but you can call the constructor of the parent class by calling `super()`.
     - Constructor and a method they both run a block of code but the difference is in calling them.
-    - Constructor gets called automatically when we create an object of that class, we cannot call it explicitly
-      though we can call parent class constructor using 'super' keyword.
-    - We call method directly using their name.
+    - We can call method directly using their name.
     - Constructor Syntax -         
               public class SomeClass{
                 SomeClass(parameter_list){ 
                   ...
                 } 
               ...
               }
-Note: 
-Constructor name is the same as class name
-Constructor does not have return type
-
+    - Note: 
+        Constructor name is the same as class name
+        Constructor does not have return type
+        
```

Comment:
use this for **note**:
| Note: |
| :--- |
| In the above syntax, the name of the constructor is the same at that of class and it has no return type. |